package.json

```diff
@@ -3,8 +3,8 @@
   "name": "privly",
   "version": "0.5.0",
   "description": "Privly enables you to post and read private content through any website without those websites storing or reading your content. This extension is still experimental. For more information on Privly development, go to https://priv.ly/pages/develop",
-  "main": "lib/extension.js",
-  "author": "Sean McGregor, Daniel Reichert, Hitesh Sharma",
```

Comment:
Initially, I added all the contributors to that list. So I thought if the contributors would increase, that list would grow and would look a bit strange. So I added details [here](https://github.com/privly/privly-jetpack/pull/11/files#diff-04c6e90faac2675aa89e2176d2eec7d8R22) instead. 
I just thought one name looks good for an author :P.
We could put `The Privly Foundation` like how Chrome has it ? Or I can put both of our names back, no problemo.